ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpTask.java

```diff
@@ -182,11 +183,14 @@ private Long bootStrapDump(Path dumpRoot, DumpMetaData dmd, Path cmRoot) throws
 
       Path dbRoot = dumpDbMetadata(dbName, dumpRoot);
       dumpFunctionMetadata(dbName, dumpRoot);
+
+      Utils.setDbBootstrapDumpState(hiveDb, dbName, Utils.ReplDumpState.ACTIVE.toString());
       for (String tblName : Utils.matchesTbl(getHive(), dbName, work.tableNameOrPattern)) {
         LOG.debug(
             "analyzeReplDump dumping table: " + tblName + " to db root " + dbRoot.toUri());
         dumpTable(dbName, tblName, dbRoot);
       }
+      Utils.setDbBootstrapDumpState(hiveDb, dbName, Utils.ReplDumpState.IDLE.toString());
```

Comment:
Currently, i believe, we don't have this use case of multiple replica clusters for Beta release. Also, this fix for blocking rename during bootstrap dump is temporary and will be removed before GA. So, need not put any complex fix right now. 
Also, as Thejas mentioned, Rename is very rare operation and hence won't impact much.